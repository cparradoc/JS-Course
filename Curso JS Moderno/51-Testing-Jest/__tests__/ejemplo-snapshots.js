const cliente = {
    nombre: 'Carlos 2',
    balance: 150000,
    tipo: 'Premium'
};

//npm t -- -u para sobreescribir el snapshot


describe('Testing al cliente', () => {
    test('Es Carlos', () => {
        expect(cliente).toMatchSnapshot();
    });
});