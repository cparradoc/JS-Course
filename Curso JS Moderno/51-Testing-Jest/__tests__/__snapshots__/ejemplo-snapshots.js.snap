// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al cliente Es Carlos 1`] = `
Object {
  "balance": 150000,
  "nombre": "Carlos 2",
  "tipo": "Premium",
}
`;
